ec2/sc-ec2-linux-jumpcloud-workflows-development.yaml

```diff
@@ -0,0 +1,496 @@
+                  - "--tags Key=Name,Value=$EC2_INSTANCE_ID-root Key=cloudformation:stack-name,Value=$STACK_NAME "
+                  - "Key=Department,Value=$DEPARTMENT Key=Project,Value=$PROJECT Key=OwnerEmail,Value=$OWNER_EMAIL"
+        TagInstance:
+          files:
+            /opt/sage/bin/apply_name_tag.sh:
+              content: |
+                #! /bin/bash
+
+                set -xe
+
+                source /opt/sage/bin/instance_env_vars.sh
+                RESOURCE_ID=${STACK_ID##*/}
+                PRODUCTS=$(/usr/bin/aws --region $AWS_REGION \
+                  servicecatalog search-provisioned-products \
+                  --filters SearchQuery=$RESOURCE_ID )
+                NUM_PRODUCTS=$(echo $PRODUCTS | jq '.TotalResultsCount')
+                if ["$NUM_PRODUCTS" -ne 1]
+                then
+                  echo "ERROR: there are $NUM_PRODUCTS provisioned products, cannot isolate a name for tagging."
+                  exit 1
+                fi
+                NAME=$(echo $PRODUCTS | jq '.ProvisionedProducts[0].Name')
+                /usr/bin/aws ec2 create-tags --region $AWS_REGION \
+                  --resources $EC2_INSTANCE_ID \
+                  --tags Key=Name,Value=$NAME
+              mode: "00744"
+              owner: "root"
+              group: "root"
+          commands:
+            01_name_tag:
+              command: "/opt/sage/bin/apply_name_tag.sh"
+        add_docker_user:
+          commands:
+            01_add_jc_user_to_docker_group:
+              command: "gpasswd -a $(cat /opt/sage/jcusername) docker"
+    Properties:
+      ImageId: ami-0144434f91f561456  # v1.0.0 packer-workflows AMI
+      InstanceType: !Ref 'EC2InstanceType'
+      SubnetId: !If
+        - UsePublicSubnet
+        - !ImportValue
+          'Fn::Sub': '${AWS::Region}-internalpoolvpc-PublicSubnet'
+        - !ImportValue
+          'Fn::Sub': '${AWS::Region}-internalpoolvpc-PrivateSubnet'
+      SecurityGroupIds: !If
+        - UsePublicSubnet
+        - [!Ref PrivateSubnetSecurityGroup, !Ref PublicSubnetSecurityGroup]
+        - [!Ref PrivateSubnetSecurityGroup]
+      KeyName: 'scipool'
+      BlockDeviceMappings:
+        -
+          DeviceName: "/dev/sda1"
+          Ebs:
+            DeleteOnTermination: true
+            VolumeSize: !Ref VolumeSize
+            Encrypted: true
+      IamInstanceProfile: !Ref 'InstanceProfile'
+      UserData:
+        Fn::Base64: !Sub |
+          #!/bin/bash
+          export DEBIAN_FRONTEND=noninteractive
+          /usr/bin/apt-get update -y
+          /usr/bin/apt-get install -y python-pip
+          /usr/bin/apt-get install -y python-setuptools
+          /bin/mkdir -p /opt/aws/bin /opt/sage/bin
+          /usr/bin/python2.7 /usr/lib/python2.7/dist-packages/easy_install.py --script-dir /opt/aws/bin https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz
```

Comment:
This was necessary after make python 3 the default in the base image.

Reply:
hmm, what do you think about adding this to the base image?

Reply:
I knew you would make that suggestion. :) Might as well.